kitsune/settings.py

```diff
@@ -641,7 +641,6 @@ def _username_algo(email):
     "mozilla_django_oidc",
     "corsheaders",
     "kitsune.users",
-    "authority",  # Remove after its migrations have been reverted.
```

Comment:
Let me talk to @escattone about that - his replacement efforts may already have a plan for that in place.  If not it might make sense to have a separate issue opened for that piece.